pkg/scheduler/framework/v1alpha1/registry.go

```diff
@@ -68,19 +68,3 @@ func (r Registry) Unregister(name string) error {
 	delete(r, name)
 	return nil
 }
-
-// NewRegistry builds a default registry with all the default plugins.
-// This is the registry that Kubernetes default scheduler uses. A scheduler that
-// runs custom plugins, can pass a different Registry and when initializing the
-// scheduler.
-func NewRegistry() Registry {
```

Comment:
A lot of places use it.

Reply:
yeah, I will see how the various components fail and updated them.